sdcm/sct_config.py

```diff
@@ -1205,6 +1205,20 @@ class SCTConfiguration(dict):
         dict(name="data_volume_disk_iops", env="SCT_DATA_VOLUME_DISK_IOPS",
              type=int,
              help="Number of iops for ebs type io2|io3|gp3"),
+
+        dict(name="nemesis_filter", env="SCT_NEMESIS_FILTER",
```

Comment:
I would call it `nemesis_include_filter`

Reply:
At first I called it include_list, but I think it's more confusing.
Think about it, usually when you filter you "select" the things you would like to see.